src/components/Forms/MaskedTextField.js

```diff
@@ -91,78 +91,80 @@ const getInputSyles = ({ props, theme, isFocused }) => {
   }
 }
 
+export const propTypes = {
```

Comment:
Personally I would give this a more specific name since it's being exported. It's a more scalable approach taking IDE autocomplete features into consideration.

Reply:
Went with a more specific naming convention.